torch/fx/experimental/symbolic_shapes.py

```diff
@@ -2388,6 +2982,15 @@ def hint(s):
 
         if not _simplified:
             for source, expr in input_guards:
+                if _translation_validator_enabled():
+                    # Ignore sources that were not turned into SymInts.
+                    srcname = source.name()
+                    if srcname in self.source_to_symbol:
+                        r = sympy.Eq(self.source_to_symbol[srcname], expr)
+                        self._add_output_guard(r)
+                    elif srcname not in concrete_sources:
+                        self.log.warning("source not found: %s (== %s)", srcname, expr)
```

Comment:
Temporar warnings for identifying cases when: there is a source that reaches `produce_guards`, but does not go through `create_symintnode`.